tf_module/variables.tf

```diff
@@ -0,0 +1,32 @@
+variable "monitor_log_group_name" {
+    description = "The name of the log group which should be sent to Papertrail"
+}
+
+variable "monitor_log_group_arn" {
+    description = "The ARN of the log group which should be sent to Papertrail"
+}
+
+variable "papertrail_host" {
+    description = "FQDN or IP of the Papertrail service endpoint"
+}
+
+variable "papertrail_port" {
+    description = "The endpoint TCP port"
+}
+
+variable "filter_pattern" {
+    description = "A CloudWatch Logs filter pattern for the log group which is subscribed to"
+}
+
+variable "timeout" {
+    description = "The timeout for the Lambda which sends log entries to Papertrail"
+    default = "10"
+}
+
+variable "lambda_log_role_arn" {
+    description = "The ARN for the role used by the Lambda which sends logs to Papertrail. Must include permissions for writing to CloudWatch logs."
```

Comment:
@uhef I respond to your comment here in the context of a PR instead to make the discussion easier to follow.

> Shouldn't this role need read access to CloudWatch logs and capability to describe and use network interfaces (in order to relay logs to papertrail)

1. The Lambda itself does not read from CloudWatch. Instead a subscription is created for the log group which invokes the Lambda instead. So the required permission is for CloudWatch (events.amazonaws.com) to be able to invoke the function and pass its data to it.
2. My understanding is that Lambdas only need the `ec2:createNetworkInterface` (and similar permissions) when they need to create an interface for joining a VPC. By default Lambdas should be "on the public Internet" so if they are accessing publicly accessible services like Papertrail no additional permissions should be necessary.
3. The permission to write to CloudWatch is necessary since the Lambda that sends to Papertrail might _itself_ throw errors in the case on unexpected input for example.